content/messaging/agent-chat-api/rtm-reference/index.mdx

```diff
@@ -4493,7 +4493,7 @@ New positions and wait times for queued chats.
 
 ## Possible errors
 
-<Errors errors="authentication,authorization,internal,license_expired,pending_requests_limit_reached,requester_already_offline,requester_awaiting_approval,request_timeout,requester_suspended,validation,wrong_product_version,seats_limit_exceeded,chat_inactive,too_many_requests"/>
+<Errors errors="authentication,authorization,internal,license_expired,pending_requests_limit_reached,requester_already_offline,requester_awaiting_approval,request_timeout,requester_suspended,validation,wrong_product_version,seats_limit_reached,chat_inactive,too_many_requests"/>
```

Comment:
Bamboo failed on this file. I checked it and it looks like you forgot to add `seats_limit_reached` to src > constant > index.js > ERRORS

Reply:
ups, not exactly, it must have changed when I did bulk rename of `limit_exceeded` to `limit_reached`

Reply:
could be :)